packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -669,6 +722,13 @@ describe('ModelInstanceDocumentHandler', () => {
     )
   })
 
+  test('throws error when applying genesis commit with invalid length', async () => {
+    ModelInstanceDocument.MAX_DOCUMENT_SIZE = 10
+    await expect(
+      ModelInstanceDocument._makeGenesis(context.api, { myData: 'abcdefghijk' }, METADATA)
```

Comment:
when we do the follow-up to clean up the tests, this should really be testing that the commit fails to apply in `applyCommit`, not in makeGenesis.  So we can set the doc size big, make the commit, then set it small and try to apply it and it should fail.